pkg/cloudprovider/provider/openstack/provider.go

```diff
@@ -439,6 +439,7 @@ func (p *provider) AddDefaults(spec clusterv1alpha1.MachineSpec) (clusterv1alpha
 	return spec, nil
 }
 
+//nolint:gocyclo
```

Comment:
This is not the appropriate fix for this. Please add an exception for this in `golangci.yaml`.

Let me know if you need assistance with that

Reply:
ok, thanks! :)